### ISSUE TITLE
[Feat]: ubuntu any version systemd service not applicable to reload

### ISSUE THREAD

**Issue by zyxep** [netdata/netdata#13743]
### Bug description

Hi,

I am trying to trigger a `systemctl reload netdata` when updating some kind of config files.
But i get `Failed to reload netdata.service: Job type reload is not applicable for unit netdata.service.`

### Expected behavior

What about adding that functionality or alias for stop/start/restart ?

### Steps to reproduce

triggering systemctl reload netdata

### Installation method

kickstart.sh

### System info

```shell
Linux gct-dia-srv-internal-001 5.4.0-124-generic #140-Ubuntu SMP Thu Aug 4 02:23:37 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
/etc/lsb-release:DISTRIB_ID=Ubuntu
/etc/lsb-release:DISTRIB_RELEASE=20.04
/etc/lsb-release:DISTRIB_CODENAME=focal
/etc/lsb-release:DISTRIB_DESCRIPTION="Ubuntu 20.04.5 LTS"
/etc/os-release:NAME="Ubuntu"
/etc/os-release:VERSION="20.04.5 LTS (Focal Fossa)"
/etc/os-release:ID=ubuntu
/etc/os-release:ID_LIKE=debian
/etc/os-release:PRETTY_NAME="Ubuntu 20.04.5 LTS"
/etc/os-release:VERSION_ID="20.04"
/etc/os-release:VERSION_CODENAME=focal
/etc/os-release:UBUNTU_CODENAME=focal
```

### Netdata build info

```shell
$ netdata -W buildinfo
Version: netdata v1.36.0-158-nightly
Configure options:  '--build=x86_64-linux-gnu' '--includedir=${prefix}/include' '--mandir=${prefix}/share/man' '--infodir=${prefix}/share/info' '--disable-silent-rules' '--libdir=${prefix}/lib/x86_64-linux-gnu' '--libexecdir=${prefix}/lib/x86_64-linux-gnu' '--disable-maintainer-mode' '--prefix=/usr' '--sysconfdir=/etc' '--localstatedir=/var' '--libdir=/usr/lib' '--libexecdir=/usr/libexec' '--with-user=netdata' '--with-math' '--with-zlib' '--with-webdir=/var/lib/netdata/www' '--disable-dependency-tracking' 'build_alias=x86_64-linux-gnu' 'CFLAGS=-g -O2 -fdebug-prefix-map=/usr/src/netdata=. -fstack-protector-strong -Wformat -Werror=format-security' 'LDFLAGS=-Wl,-Bsymbolic-functions -Wl,-z,relro' 'CPPFLAGS=-Wdate-time -D_FORTIFY_SOURCE=2' 'CXXFLAGS=-g -O2 -fdebug-prefix-map=/usr/src/netdata=. -fstack-protector-strong -Wformat -Werror=format-security'
Install type: binpkg-deb
    Binary architecture: x86_64
    Packaging distro:
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              YES
    ACLK:                       YES
    TLS Host Verification:      YES
    Machine Learning:           YES
    Stream Compression:         YES
Libraries:
    protobuf:                YES (system)
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  NO
    libcrypto:               YES
    libm:                    YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    YES
    EBPF:                    YES
    IPMI:                    YES
    NFACCT:                  YES
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 NO
    Prometheus Remote Write: YES
```

**Comment by ilyam8** [comment]
Hi, @zyxep. I don't think that is a bug. Why do you think we need an alias for restart (reload)?

**Comment by zyxep** [comment]
well, either the reload functionality should be there.
e.g. `kill -HUP` to reload the process, so reading the configs again.

Or an alias for reload that trigger a restart, when you do changes config files.
I discovered this when writing our automation for netdata in saltstack.
Saltstack in our case couldn't trigger a `systemd reload netdata`

It could be a feature, but i chose to create it as a bug since it's quite normal to have reload option in systemd.

**Comment by ilyam8** [comment]
Ok, I see. Anyway, I do not think the lack of reloading functionality can be considered a bug.

> Saltstack in our case couldn't trigger a systemd reload netdata

But I guess it is not a big deal to change to `systemd restart netdata`.

cc @Ferroin @cakrit

**Comment by cakrit** [comment]
I'd rather we actually support `reload` and have it be whatever is possible at any time. Right now, it can only reload the health configuration, but we may change that in the future. So I wouldn't make it equivalent to restart. 
I agree it's not a bug ,even if it's widely available. It could actually be a bit misleading to say that we successfully executed the reload command and yet the only config changes we applied are related to alerts. So until we have proper reload capabilities for almost every config setting, I'd consider it a feature request.
